### PR TITLE
Name ocs subscription the same like done by UI

### PR DESCRIPTION
**Pull request by petr-balogh**
In UI deployment the subscription is called ocs-operator,
so we should follow the same naming convention.

Fixes: #2478

Signed-off-by: Petr Balogh <pbalogh@redhat.com>

### CODE REVIEW

**Comment by petr-balogh** [comment]
We cannot merge this if we are running any upgrade job.  Than if someone would like to upgrade existing cluster deployed via ocs-ci some time before the merge they will need to change the name of the subscription before running the upgrade on the cluster.

**Review by vavuthu — APPROVED** [review]

**Comment by petr-balogh** [comment]
Adding WIP to prevent this to be merged. We will merge once no upgrade job will run

**Review by dahorak — APPROVED** [review]

**Pull request merged**
